### ISSUE TITLE
Unknown compiler option '-c'.

### ISSUE THREAD

**Issue by foreggs** [alexjoverm/typescript-library-starter#63]
After running `yarn install` and `yarn start`, this is the error i get:
```
> tsc-watch --onSuccess 'rollup -c'
error TS5023: Unknown compiler option '-c''.
```

**Comment by foreggs** [comment]
I think I found the cause of the problem. After removing the `-c` flag, command 'rollup' is not recognised....
I am using windows and linux alongside and it seems to work on linux. 

After replacing `"tsc-watch --onSuccess 'rollup'"` with `"tsc-watch --onSuccess \"rollup\""` it suddenly started working.

**Comment by alexjoverm** [comment]
Hi @foreggs! Is working here for me in MacOs as well... Could you send a PR with the solution that works on Windows as well? Also, if you can check if the rest of the commands are working on Windows that'd be great :)

**Comment by foreggs** [comment]
These commands don't work on windows:
```json
 {
    "start": "tsc-watch --onSuccess 'rollup -c'",
    "report-coverage": "cat ./coverage/lcov.info | coveralls"
  },
```
And here is how to fix that:
```json
 {
    "start": "tsc-watch --onSuccess \"rollup -c\""
  },
```

When it comes to the 'cat' command, there is the 'type' command on windows. There is no cross platform equivalent. I'm not familiar with coveralls so I can't help you with that one.

**Comment by alexjoverm** [comment]
The report coverage is meant to run only on a CI server (i.e.: Travis)  so I'm not worried about that one.

Thanks for your help ;)

**Comment by foreggs** [comment]
#65

**Comment by alexjoverm** [comment]
Many many thanks! Closed by #65
